CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 56 Release Notes
 
+### Retry with More Memory as workflow option
+
+Retry with more memory is now customizable via a global config. The multiply factor is passed in through the workflow 
+options. More details can be found [here](https://cromwell.readthedocs.io/en/develop/wf_options/Overview.md#retry-with-more-memory-multiplier).
+
```

Comment:
```suggestion
### Retry with More Memory

The experimental memory retry feature gains per-workflow customization and includes breaking changes:
- The per-backend configuration key `<backend>.config.memory-retry.error-keys` has been removed and replaced with global key `system.memory-retry-error-keys`
- The per-backend configuration key `<backend>.config.memory-retry.multiplier` has been replaced with *workflow option* `memory_retry_multiplier`

More details can be found [here](https://cromwell.readthedocs.io/en/develop/wf_options/Overview.md#retry-with-more-memory-multiplier).
```

Reply:
I figured it was worth writing this out even just for our team to use as reference when we help customers upgrade

Reply:
This is much better! Thank you!